packages/edit-navigation/src/index.js

```diff
@@ -31,6 +32,43 @@ function disableInsertingNonNavigationBlocks( settings, name ) {
 	return settings;
 }
 
+function removeNavigationBlockSettingsUnsupportedFeatures( settings, name ) {
+	if ( name !== 'core/navigation' ) {
+		return settings;
+	}
+
+	return {
+		...settings,
+		supports: {
+			...omit( settings.supports, [
+				'anchor',
+				'customClassName',
+				'__experimentalColor',
+				'__experimentalFontSize',
+			] ),
+			customClassName: false,
+		},
+	};
+}
+
+const removeNavigationBlockEditUnsupportedFeatures = createHigherOrderComponent(
+	( BlockEdit ) => ( props ) => {
+		if ( props.name !== 'core/navigation' ) {
+			return <BlockEdit { ...props } />;
+		}
+
+		return (
+			<BlockEdit
+				{ ...props }
+				hasSubmenuIndicatorSetting={ false }
+				hasItemJustificationControls={ false }
+				hasListViewModal={ false }
+			/>
+		);
+	},
+	'removeNavigationBlockEditUnsupportedFeatures'
+);
```

Comment:
Ditto https://github.com/WordPress/gutenberg/pull/25329/files#r490875716

Reply:
I don't agree as there are more cases where the setting needs to be `true` (post editor, site editor) than there are where it needs to be `false` (navigation editor). There are possibilities for mistakes either way.